recipes/mosec/build.sh

```diff
@@ -0,0 +1,12 @@
+#!/usr/bin/env bash
+
+set -euo pipefail
+
+cargo build --release
+mkdir -p mosec/bin
+cp target/$CARGO_BUILD_TARGET/release/mosec mosec/bin
+pip install .
```

Comment:
```suggestion
$PYTHON -m pip install . --no-build-isolation \
    --no-deps --ignore-installed --no-index --no-cache-dir -vv
```

Reply:
Thanks for your advice. I have confirmed that the latest commit can handle this. Just need to copy the binary file to the installed package directory and conda will pack everything under that directory. PTAL.